script/setup/newProject/tokenInfo.ts

```diff
@@ -0,0 +1,255 @@
+      message: `Enter initial supply owner:`,
+      validate: (value) => validateEmptyValue(String(value).trim()),
+      initial: ownerAddress,
+    },
+  ]);
+  superTokenInfo = { ...superTokenInfo, ...info };
+  superTokenInfo.symbol = superTokenInfo.symbol.toUpperCase();
+  if (isFreshDeployment) {
+    let response = await prompts([
+      {
+        name: "initialSupply",
+        type: "text",
+        message: `Enter initial supply(enter formatted value, ex: 1000 for 1000 WETH) :`,
+        validate: (value) => validateEmptyValue(String(value).trim()),
+      },
+      {
+        name: "initialChain",
+        type: "select",
+        message:
+          "Select the chain where this initial supply is to be minted (to mint on multiple chains, contact team)",
+        choices: controllerChains.map((chain) => {
+          return {
+            title: chainSlugReverseMap.get(String(chain)),
+            value: chain,
+          };
+        }),
+      },
+    ]);
+    superTokenInfo.initialSupply = response.initialSupply;
+    superTokenInfo.initialChain = response.initialChain;
+  } else {
+    while (true) {
+      let response = await prompts([
+        {
+          name: "chain",
+          type: "select",
+          message: "Select the chain where the token is already deployed",
+          choices: [...vaultChains, ...controllerChains].map((chain) => {
+            return {
+              title: chainSlugReverseMap.get(String(chain)),
+              value: chain,
+            };
+          }),
+        },
+        {
+          name: "address",
+          type: "text",
+          message:
+            "Enter the address of the deployed token on the selected chain",
+          validate: (value) => validateEthereumAddress(value.trim()),
+        },
+        {
+          name: "addMoreTokens",
+          type: "confirm",
+          message:
+            "Want to add more token addresses? If a token is already deployed somewhere, script won't deploy again.",
+        },
+      ]);
+
+      superTokenInfo.currentAddresses = superTokenInfo.currentAddresses ?? [];
+      superTokenInfo.currentAddresses.push({
+        token: superTokenInfo.symbol.toUpperCase(),
+        chainSlug: response.chain,
+        address: response.address,
+      });
+      if (!response.addMoreTokens) break;
+    }
+  }
+  return superTokenInfo;
+};
+
+export const getMergeInboundWithTokens = async (tokens: Tokens[]) => {
+  const mergeInboundWithTokens: {
+    [key in Tokens]?: Tokens[];
+  } = {};
+
+  const isEthAndWethPresent =
+    tokens.includes(Tokens.ETH) && tokens.includes(Tokens.WETH);
+  const isUsdcAndUsdcePresent =
+    tokens.includes(Tokens.USDC) && tokens.includes(Tokens.USDCE);
+
+  if (isEthAndWethPresent) {
+    const response = await prompts([
+      {
+        name: "mergeEthAndWeth",
+        type: "confirm",
+        message:
+          "Want to merge ETH and WETH deposits to app chain? Both deposits will mint WETH on app chain",
+      },
```

Comment:
We hit a bad case here when someone deploys on a chain where eth is not the gas token.

Reply:
if eth is not gas token, then he wont be bridging eth right? only weth will be bridged, where this case wont reach.